### PR TITLE
improve collapsible form code

### PR DESCRIPTION
**Pull request by sarayfg**
Worked in `collapsibles_form.js` :
reduce functions in just one using event.currentTarget and querySelectorAll.
We create the class "panel" to make it works.

Let me know if there is any doubt.

### CODE REVIEW

**Comment by sarayfg** [comment]
there's a layout's problem in fill-in section to improve

File: _src/assets/js/collapsible_form.js [hunk 1]
```diff
@@ -1,81 +1,17 @@
...
-        sectionDesign.classList.add('hidden');
-        arrowIconDesign.classList.toggle('fa-chevron-up');
-        arrowIconDesign.classList.toggle('fa-chevron-down');
-    }
-    else if (!sectionFill.classList.contains('hidden')) {
-        sectionFill.classList.add('hidden');
-        arrowIconFill.classList.toggle('fa-chevron-up');
-        arrowIconFill.classList.toggle('fa-chevron-down');
-    }
+function handlerLegendClick(event){
```
Comment (valerialp) [inline _src/assets/js/collapsible_form.js]: we could do the function as arrow function :)

Comment (sarayfg) [reply _src/assets/js/collapsible_form.js]: solved :)

**Comment by valerialp** [comment]
@sarayfg Well job!! you can merge ;)

**Review by valerialp — APPROVED** [review]

**Pull request merged**